helm/alfresco-content-services/templates/ingress-repository.yaml

```diff
@@ -50,10 +52,28 @@ spec:
   {{- end }}
       paths:
       - path: {{ .Values.repository.ingress.path }}
+        {{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.GitVersion }}
+        pathType: Prefix
+        backend:
+          service:
+            Name: {{ template "content-services.shortname" . }}-repository
```

Comment:
name must be lower-case!

Reply:
Yes, this PR contains 2 commits and it's already fixed in the second commit.